src/server/terrain/simple_structures/SimpleTreeModel.zig

```diff
@@ -57,37 +56,58 @@ pub fn loadModel(arenaAllocator: NeverFailingAllocator, parameters: ZonElement)
 	return self;
 }
 
+fn getWoodBlock(self: *SimpleTreeModel, data: u16) main.blocks.Block {
+	if(self.woodBlock.mode() == main.rotation.getByID("log")) {
```

Comment:
Oh nice, you fixed the problem of stair branches in the glass forest.
For completeness, could you also apply this fix to the fallen tree?